ccloud-monitoring/docs/consumer-scenarios/increasing-consumer-lag.rst

```diff
@@ -0,0 +1,168 @@
+.. _ccloud-monitoring-consumer-increasing-consumer-lag:
+
+Increasing Consumer Lag
+***********************
+
+Consumer lag is a tremendous performance indicator. It tells you the offset difference between the
+producer's last produced message and the consumer group's last commit. If you are unfamiliar with
+consumer groups or concepts like committing offsets, please refer to this
+`Kafka Consumer documentation <https://docs.confluent.io/platform/current/clients/consumer.html>`__.
+
+A large consumer lag, or a quickly growing lag, indicates that the consumer is not able to keep up with
+the volume of messages on a topic.
+
+This scenario will look at |ccloud| metrics from the Metrics API, `kafka-lag-exporter <https://github.com/lightbend/kafka-lag-exporter>`__ metrics, and
+client metrics from the client application’s MBean object ``kafka.consumer:type=consumer-fetch-manager-metrics,client-id=<client_id>``.
+
+Introduce failure scenario
```

Comment:
It's not really a failure scenario, it's misleading to call it that.

Reply:
I would argue that teams do have SLAs to process the latest messages within a period of time, so having a lag (if big enough) would could cause you to break that SLA. Imagine going from 25 second delays to 12 minute delays (possible if consumer code is tweaked), that could be a failure to meet your SLA of processing messages within a 30 second delay. 
It's definitely true that this lag introduced here isn't that massive. I can add a back story in the intro about how this could turn into a failure scenario but with resource restrictions, we will show a "degraded state" (barely). Or change the title to "Introduce degraded state scenario" (imo this is still misleading given the small lag that is introduced).